packages/inter-protocol/src/psm/psm.js

```diff
@@ -134,6 +135,41 @@ export const start = async (zcf, privateArgs, baggage) => {
   };
   updateMetrics();
 
+  /** @param {Brand<'nat'>} b */
+  const amtShape = b => harden({ brand: b, value: M.nat() });
```

Comment:
there's a helper for this already, `makeNatAmountShape`. It takes an optional `min` parameter, but I don't think that's helpful here.

This function is getting called repeatedly in multiple blocks, so please save the constants here to re-use
```suggestion
  const AnchorAmountShape = makeNatAmountShape(stableBrand);
  const StableAmountShape = makeNatAmountShape(anchorBrand);
```